reconcile/gitlab_housekeeping.py

```diff
@@ -21,14 +22,17 @@
 DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
 
 
-def clean_pipelines(dry_run, gl_project, pipelines, pipeline_timeout=60):
+def get_timeout_pipelines(pipelines: List[Any],
```

Comment:
Your pipelines look like `Dict` to me.

Reply:
Also, naming nit, this sounds like you're getting the time out parameter for the pipelines instead of the ones that have timed out.
```suggestion
def get_timed_out_pipelines(pipelines: List[Dict],
```